### PR TITLE
East African

### PR DESCRIPTION
**Pull request by Gr770**

### CODE REVIEW

File: common/flavorization/00_ctp_titles_02.txt [hunk 1]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	cultures = { ethiopian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+}
+title_prince_male_ethiopian_empire = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = empire
+	priority = 130
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3513]: This should be higher prio than title_prince_male_ethiopian (Kingdom one)

File: common/flavorization/00_ctp_titles_02.txt [hunk 2]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+	only_holder = yes
+}
+kingdom_feudalclan_ethiopian = {
+	type = title
+	tier = kingdom
+	priority = 143
+	cultures = { ethiopian }
+	governments = { feudal_government clan_government republic_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3446]: Remove republic here

File: common/flavorization/00_ctp_titles_02.txt [hunk 3]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+duchy_feudalclan_ethiopian_independent = {
+	type = title
+	tier = duchy
+	priority = 124
+	cultures = { ethiopian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+	only_independent = yes
+}
+king_feudalclan_male_ethiopian = {
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3410]: No feudal clan for kings

Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt]: Done

File: common/flavorization/00_ctp_titles_02.txt [hunk 4]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+###############################################
+
+baron_male_nubian = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 103
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3559]: add tribal

File: common/flavorization/00_ctp_titles_02.txt [hunk 5]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+}
+baron_female_holder_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = barony
+	priority = 104
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3579]: tribal

File: common/flavorization/00_ctp_titles_02.txt [hunk 6]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+	only_holder = yes
+}
+barony_nubian = {
+	type = title
+	tier = barony
+	priority = 103
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3588]: tribal

File: common/flavorization/00_ctp_titles_02.txt [hunk 7]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+}
+count_female_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 113
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3608]: This should not have tribal?

File: common/flavorization/00_ctp_titles_02.txt [hunk 8]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 123
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+duke_female_nubian = {
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3640]: This should not have tribal? And also add in feudal clan

File: common/flavorization/00_ctp_titles_02.txt [hunk 9]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+}
+duke_feudalclan_male_nubian_independent = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 125
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3676]: No tribal

File: common/flavorization/00_ctp_titles_02.txt [hunk 10]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 126
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+	only_independent = yes
+}
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3690]: No tribal

File: common/flavorization/00_ctp_titles_02.txt (hunk 8 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt:3640]: This is the generic term for lady

File: common/flavorization/00_ctp_titles_02.txt [hunk 11]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+duchy_feudalclan_nubian_independent = {
+	type = title
+	tier = duchy
+	priority = 125
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3696]: no tribal

File: common/flavorization/00_ctp_titles_02.txt [hunk 12]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+}
+emperor_female_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = 153
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3745]: add republic

File: common/flavorization/00_ctp_titles_02.txt (hunk 7 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt:3608]: This is the generic term for lady

File: common/flavorization/00_ctp_titles_02.txt [hunk 13]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+}
+empire_nubian = {
+	type = title
+	tier = kingdom
+	priority = 153
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3753]: add republic

File: common/flavorization/00_ctp_titles_02.txt [hunk 14]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	top_liege = no
+}
+emperor_male_nubian = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 153
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3735]: Add republic

File: common/flavorization/00_ctp_titles_02.txt (hunk 8 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt:3640]: The ruler terms are gender neutral.

File: common/flavorization/00_ctp_titles_02.txt (hunk 14 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt:3735]: Done and I fixed the kingdom

File: common/flavorization/00_ctp_titles_02.txt (hunk 12 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt:3745]: Done and I fixed the kingdom

File: common/flavorization/00_ctp_titles_02.txt [hunk 15]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 113
+	cultures = { nubian }
+	governments = { tribal_government }
+	top_liege = no
+}
+count_female_tribal_holder_nubian = {
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3856]: Female is missing. Please look into this again

File: common/flavorization/00_ctp_titles_02.txt [hunk 16]
```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
...
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 123
+	cultures = { nubian }
+	governments = { tribal_government }
+	top_liege = no
+}
+duke_female_tribal_holder_nubian = {
```
Comment (Charanraja123) [inline common/flavorization/00_ctp_titles_02.txt:3885]: Female is missing. Please look into this again

File: common/flavorization/00_ctp_titles_02.txt (hunk 11 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt]: Fixed

File: common/flavorization/00_ctp_titles_02.txt (hunk 10 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt:3690]: Fixed

File: common/flavorization/00_ctp_titles_02.txt (hunk 9 above)
Comment (Gr770) [reply common/flavorization/00_ctp_titles_02.txt]: Fixed

**Pull request merged**